src/app/containers/Brand/index.jsx

```diff
@@ -1,11 +1,24 @@
-import React, { useContext } from 'react';
+import React, { Fragment, useContext } from 'react';
 import Brand from '@bbc/psammead-brand';
 import { ServiceContext } from '../../contexts/ServiceContext';
 
 const BrandContainer = () => {
   const { brandName, brandSVG } = useContext(ServiceContext);
+  const defaultBrandName = 'BBC News';
+  let localisedBrandName;
 
-  return <Brand brandName={brandName} svg={brandSVG} />;
+  if (brandName !== defaultBrandName) {
+    localisedBrandName = (
+      <Fragment>
+        <span lang="en_GB">{defaultBrandName}</span>
+        {brandName.substring(
+          brandName.indexOf(defaultBrandName) + defaultBrandName.length,
+        )}
+      </Fragment>
+    );
+  }
+
+  return <Brand brandName={localisedBrandName || brandName} svg={brandSVG} />;
```

Comment:
`Brand` expects `brandName` to be a string but `localizedBrandName` is returning a `node`